### DISCUSSION TITLE
Deleting a source member

### DISCUSSION THREAD

**Discussion by Gregory1701** [codefori/vscode-ibmi#1298] [IBM i]
Hey guys!  I created a source member in my source physical file and no longer need it.  When I right click on the member and click Delete Member, I get the confirmation toast and click yes. 
![image](https://github.com/halcyon-tech/vscode-ibmi/assets/132368707/2dea6a30-0112-408d-9659-5a5b6344c0aa)
But then I get this error message:
![image](https://github.com/halcyon-tech/vscode-ibmi/assets/132368707/16bb0972-2ef7-46cb-b8eb-3ce74ae424f2)
Am I somehow doing something wrong?  Or is this needing a fix?

**Comment by sebjulliand** [comment]
Hi @Gregory1701,
I gave it a try and it is indeed broken in version 1.8.0. But it's been fixed in the meantime (by an unrelated PR 😅) as the issue is gone when running with the latest build of the master branch.
So it will be fixed as soon as a new release will be out (poke @worksofliam).

**Reply by Gregory1701** [reply]
Okay, thanks for checking this out.  Much appreciated!
